modules/Microsoft.Network/virtualNetworks/deploy.bicep

```diff
@@ -19,6 +19,23 @@ param ddosProtectionPlanId string = ''
 @description('Optional. Virtual Network Peerings configurations.')
 param virtualNetworkPeerings array = []
 
+@description('Optional. Indicates if VM protection is enabled for all the subnets in the virtual network.')
+param enableVmProtection bool = false
```

Comment:
Should we set this to true by default? Sounds like a meaningful security setting

Reply:
Seeing this ==> https://github.com/Azure/azure-cli/issues/14059#issuecomment-667289923

I want to remove it now :D

Reply:
Agreed

Reply:
Updated, tested locally and regenerated doco